src/core_plugins/kibana/server/lib/export/collect_panels.js

```diff
@@ -0,0 +1,30 @@
+import { get } from 'lodash';
+
+import collectIndexPatterns from './collect_index_patterns';
+import collectSearchSources from './collect_search_sources';
+
+export const deps = {
+  collectSearchSources,
+  collectIndexPatterns
+};
+
+export default function collectPanels(savedObjectsClient, dashboard) {
```

Comment:
Fixed this by wrapping the parse in a try catch and moving up the return to the catch statement.